src/models/Game.test.ts

```diff
@@ -223,3 +223,40 @@ describe("#isInProgress", () => {
     it("returns false", () => {});
   });
 });
+
+describe("#nextTurn", () => {
+  it("gives the turn to the next player", () => {
+    const player1 = new Player({ name: "Player 1" });
+    const player2 = new Player({ name: "Player 2" });
+    const player3 = new Player({ name: "Player 3" });
+    const players = [player1, player2, player3];
+    const game = new Game(players);
+
+    game.start();
+
+    expect(game.getCurrentPlayer()).toEqual(player1);
+
+    player1.playRandomCard(); // 5 cards in hand
+    player1.playRandomCard(); // 4 cards in hand
+    game.nextTurn();
+
+    expect(player1.getNumberOfCardsInHand()).toEqual(6);
+    expect(game.getCurrentPlayer()).toEqual(player2);
+
+    player2.playRandomCard(); // 5 cards in hand
+    player2.playRandomCard(); // 4 cards in hand
+    // player2.playRandomCard(); // 3 cards in hand
+    // player2.playRandomCard(); // 2 cards in hand
```

Comment:
🖊️  &nbsp; These will need to be re-enabled once the logic handles giving out cards until the player's hand reaches 6 cards